github/pulls_reviews.go

```diff
@@ -17,3 +20,29 @@ type PullRequestReview struct {
 	// State can be "approved", "rejected", or "commented".
 	State *string `json:"state,omitempty"`
 }
+
+func (p PullRequestReview) String() string {
+	return Stringify(p)
+}
+
+// ListReviews lists all reviews on the specified pull request.
+//
+// GitHub API docs: https://developer.github.com/v3/pulls/reviews/#list-reviews-on-a-pull-request
+func (s *PullRequestsService) ListReviews(owner string, repo string, number int) ([]*PullRequestReview, *Response, error) {
+	u := fmt.Sprintf("repos/%v/%v/pulls/%v/reviews", owner, repo, number)
+
+	req, err := s.client.NewRequest("GET", u, nil)
+	if err != nil {
+		return nil, nil, err
+	}
+
+	req.Header.Set("Accept", mediaTypePullRequestReview)
```

Comment:
Add a `// TODO: This header will be unnecessary when the API is no longer in preview.` comment above this line, for consistency.